### PR TITLE
Changes that cover a wide array

### PR DESCRIPTION
**Pull request by fbane**
- Modified core base structure for what I feel is better organized and helps when going through files
- Modified core base to separate saves between those for people who wish to share and not
- Those who wish to share their maps will have a link and if the link is past a certain time will show "Expired"
- CSS files altered for index.php for a cleaner look
- Other changes as well but the majority was to clean up the index page and uploads page.

### CODE REVIEW

**Review by ian76g — CHANGES_REQUESTED** [review]
your pull requests includes some nasty merge conflicts in unresolved forms.
please let us fix the issues first - before we merge this into the master

**Review by fbane — COMMENTED** [review]
Check and see if changes valid

File: converter.php [hunk 1]
```diff
@@ -1,9 +1,11 @@
 <?php
+
 error_reporting(E_ALL);
 ini_set('memory_limit', -1);  // just in case - previous versions had 8000x8000 px truecolor images JPEG
 set_time_limit(90);                // just in case something wents really bad -- kill script after 10 seconds
 $v = 46;                                  //version - totally not used except in next line
-//echo "\n" . 'running converter version 0.' . $v . "\n";
+
+echo "\r\n" . 'Running converter version 0.' . $v . "<br>";
```
Comment (ian76g) [inline converter.php]: has to be removed - else the auto location to the map will not work

File: converter.php [hunk 2]
```diff
@@ -93,7 +101,7 @@ function getUserIpAddr()
     ((isset($_POST['xoff']) && $_POST['xoff'] !== '') ? ($_POST['xoff']) : $bgOffsets[$bg][2]) . '" y="' .
     ((isset($_POST['yoff']) && $_POST['xoff'] !== '') ? ($_POST['yoff']) : $bgOffsets[$bg][3]) . '" width="' .
     ((isset($_POST['xsoff']) && $_POST['xoff'] !== '') ? ($_POST['xsoff']) : $bgOffsets[$bg][4]) . '" height="' .
-    ((isset($_POST['ysoff']) && $_POST['xoff'] !== '') ? ($_POST['ysoff']) : $bgOffsets[$bg][5]) . '" href="' . $bg . '" />
+    ((isset($_POST['ysoff']) && $_POST['xoff'] !== '') ? ($_POST['ysoff']) : $bgOffsets[$bg][5]) . '" href="/topo/' . $bg . '" />
             </pattern>
```
Comment (ian76g) [inline converter.php:104]: why did only one of the 4 backgrounds got relocated?

File: index.php [hunk 3]
```diff
@@ -18,37 +18,42 @@
...
+//        } else {
+//            $x=1;
+//        }
+//        if($_GET['sortorder']=='desc'){
+//            return $x;
+//        } else {
+//            return -$x;
+//        }
+//    }
+//
```
Comment (ian76g) [inline index.php:49]: This function is needed that the links on the tableheaders work.
why got it removed?

File: upload.php [hunk 4]
```diff
@@ -26,7 +26,7 @@
     }
 
 // Check file size
-    if ($_FILES["fileToUpload"]["size"] > 1500000) {
+    if ($_FILES["fileToUpload"]["size"] > 500000) {
         echo "Sorry, your file is too large.";
```
Comment (ian76g) [inline upload.php]: we need more than 500kB for larger save games

File: upload.php [hunk 5]
```diff
@@ -138,30 +156,52 @@
...
+            </div>
+          </section>
+
+          <section>
+            <h3>7. Map Options</h3>
+            <div>
+=======
             <h3>6. Slope settings</h3>
+>>>>>>> 3a151899b48d50c01198da83a8d43a2824a48f6e
             <div class="input-group input-group--row">
```
Comment (ian76g) [inline upload.php:165]: there are unresolved conflicts here...

File: upload.php [hunk 6]
```diff
@@ -138,30 +156,52 @@
...
             <div class="input-group">
-                <label for="slopeTriggerDecimals">round to X decimals</label>
+                <label for="slopeTriggerDecimals">Round to X decimals</label>
                 <input placeholder="1" type="text" id="slopeTriggerDecimals" name="slopeTriggerDecimals" value="1">
             </div>
+<<<<<<< HEAD
+          </div>
+          </section>
+=======
         </section>
```
Comment (ian76g) [inline upload.php:188]: unresolved conflicts

File: upload.php [hunk 7]
```diff
@@ -172,6 +212,7 @@
             </div>
 
 
+>>>>>>> 3a151899b48d50c01198da83a8d43a2824a48f6e
             <input class="button" type="submit" value="Upload" name="submit">
```
Comment (ian76g) [inline upload.php:196]: unresolved conflict

**Review by ian76g — CHANGES_REQUESTED** [review]

File: converter.php (hunk 2 above)
Comment (fbane) [reply converter.php:104]: Not sure on this comment. As part of the file restructure the images for the backgrounds were moved into the /topo folder so the path had to reflect that to find them.

**Review by fbane — COMMENTED** [review]
Think this should cover off everything

**Review by fbane — COMMENTED** [review]
Small changes to upload.php

File: css/main.css [hunk 8]
```diff
@@ -104,9 +104,8 @@ input[type="text"] {
   width: 100%;
   padding: 1rem 0.75rem;
   background: white;
-  border: 0;
+  border: 1px solid #AAAAAA;
   border-radius: var(--border-radius);
-  outline: 2px solid transparent;
   transition: all 0.1s ease-out;
```
Comment (JamieKDonnelly) [inline css/main.css]: @fbane removing the `outline` breaks the CSS transition on the `border` property for the `:focus` of the input

File: css/main.css [hunk 9]
```diff
@@ -301,6 +299,7 @@ tbody td {
   flex-wrap: wrap;
   justify-content: center;
   align-items: center;
+  border: 1px;
```
Comment (JamieKDonnelly) [inline css/main.css]: @fbane Not sure on what this change achieves?

File: map.template.html [hunk 10]
```diff
@@ -62,7 +62,7 @@ <h3>Edit</h3><br/>
       </div>
     </main>
 
-    <script type="text/javascript" src="svg-pan-zoom.js"></script>
-    <script type="text/javascript" src="../js/export.js"></script>
+    <script type="text/javascript" src="/topo/svg-pan-zoom.js"></script>
```
Comment (JamieKDonnelly) [inline map.template.html:65]: Just out of curiosity, why 'topo'?

**Review by JamieKDonnelly — COMMENTED** [review]
Few comments for you @fbane !

File: map.template.html (hunk 10 above)
Comment (fbane) [reply map.template.html:65]: topo as in topographic .. more specific to "maps"

File: css/main.css (hunk 8 above)
Comment (fbane) [reply css/main.css]: Hmm ok. I don't recall what I was trying here, but reverted it.

File: css/main.css (hunk 9 above)
Comment (fbane) [reply css/main.css]: Hmm ok. Reverted this one.

**Review by JamieKDonnelly — APPROVED** [review]

**Review by fbane — COMMENTED** [review]
How about now?

**Review by ian76g — APPROVED** [review]

**Pull request merged**